docs/single-cluster.md

```diff
@@ -253,7 +253,7 @@ variables:
   flux_recreate: 'false'
   gitops_ssh_url: 'git@ssh.dev.azure.com:v3/myorg/app-cluster-manifests'
   gitops_path: 'prod'
-  gitops_ssh_key: '~/cluster-deployment/keys'
+  gitops_ssh_key_path: '~/cluster-deployment/keys'
```

Comment:
Can we update this sample to have an absolute path instead of `~` notation?

Reply:
done